pkg/templates/charts/toggle/image-based-install-operator/templates/webhook-service_v1_service.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
     service.beta.openshift.io/serving-cert-secret-name: webhook-certs
   creationTimestamp: null
   name: webhook-service
```

Comment:
I think this name need to be updated

Reply:
We are pulling that from here: https://github.com/openshift/image-based-install-operator/blob/main/bundle/manifests/webhook-service_v1_service.yaml.

Reply:
:facepalm: that's my fault

But also if this was here already we may need to do that migration after all, right?

Reply:
Opened https://github.com/openshift/image-based-install-operator/pull/209, but note that that changes the file name

Reply:
File name changes should be okay. Since MCE is already deploying the `service` as `webhook-service` in previous releases, we will need to add some logic to prune that resource for the new one.